rust/src/data_structures.rs

```diff
@@ -1117,6 +1117,8 @@ pub enum OwnedWebRtcMessage {
     EmptyBinary,
 }
 
+/**
+ * TODO. Implement FBS conversion.
```

Comment:
When do you plan to do this?

Reply:
I've added a TODO item in the flatbuffers PR. It must be done for sure before merging that branch into v3.